### PR TITLE
Improve install documentation (bug 1195031)

### PR DESCRIPTION
**Pull request by Evelf**
Fix to the bug I opened: https://bugzilla.mozilla.org/show_bug.cgi?id=1195031

### CODE REVIEW

File: docs/topics/install-olympia/installation.rst [hunk 1]
```diff
@@ -34,7 +35,7 @@ The following command will install the required development files on Ubuntu or,
 if you're running a recent version, you can `install them automatically
 <apt:python-dev,python-virtualenv,libxml2-dev,libxslt1-dev,libmysqlclient-dev,memcached,libssl-dev,swig openssl,curl,libjpeg-dev,zlib1g-dev,libsasl2-dev>`_::
 
-    sudo aptitude install python-dev python-virtualenv libxml2-dev libxslt1-dev libmysqlclient-dev memcached libssl-dev swig openssl curl libjpeg-dev zlib1g-dev libsasl2-dev
+    sudo apt-get install python-dev python-virtualenv libxml2-dev libxslt1-dev libmysqlclient-dev memcached libssl-dev swig openssl curl libjpeg-dev zlib1g-dev libsasl2-dev nodejs nodejs-legacy
```
Comment (kmaglione) [inline docs/topics/install-olympia/installation.rst]: I'm not personally a fan of `aptitude`, but it does seem to be fairly popular. Is that change necessary?

File: docs/topics/install-olympia/installation.rst [hunk 2]
```diff
@@ -56,9 +57,18 @@ The following packages will get you set for olympia::
 MySQL
 ~~~~~
 
+By default, Olympia uses the root user and no password to connect to the database. If you want to change this, you could either add the database settings in your :ref:`local_settings.py <example-settings>` or set the env variable DATABASE_URL by doing::
```
Comment (kmaglione) [inline docs/topics/install-olympia/installation.rst]: Some nits:

s/uses the root user and not password to connect to the database/connects to the `olympia` database running on `localhost` as the user `root`, with no password/

s/could/can/

s/env/environment/

s/DATABASE_URL/`DATABASE_URL/

s/by doing//

File: docs/topics/install-olympia/installation.rst [hunk 3]
```diff
@@ -56,9 +57,18 @@ The following packages will get you set for olympia::
...
+
+    export DATABASE_URL mysql://<user>:<password>@<hostname>/<database>
+
 You'll probably need to :ref:`configure MySQL after install <configure-mysql>`
 (especially on Mac OS X) according to advanced installation.
 
+Elasticsearch
+~~~~~~~~~~~~~
+
+You'll need an Elasticsearch up and running during the init script. See :doc:`./elasticsearch` for more instructions.
```
Comment (kmaglione) [inline docs/topics/install-olympia/installation.rst]: s/Elasticsearch/Elasticsearch server/

**Comment by kmaglione** [comment]
These look like helpful changes. Thanks. Just some small nits.

File: docs/topics/install-olympia/installation.rst (hunk 1 above)
Comment (Evelf) [inline docs/topics/install-olympia/installation.rst]: I thought than as apt-get is installed on every Ubuntu and aptitude is not, it's may be simple for some people if we use apt-get in the instructions. If someone prefers aptitude, they could copy the package list.

Comment (kmaglione) [inline docs/topics/install-olympia/installation.rst]: On Sat, Aug 15, 2015 at 04:14:56PM -0700, Eve La Fée wrote:
>I thought than as apt-get is installed on every Ubuntu and aptitude is not, it's may be simple for some people if we use apt-get in the instructions. If someone prefers aptitude, they could copy the package list.

Fair enough.

**Comment by Evelf** [comment]
Except for the apt-get/aptitude question, all your suggestions are in the new commit :)

**Comment by kmaglione** [comment]
Great. Thanks again! r+
